From a2148ae862a454b49bc7340f2cb6b0c9d408deee Mon Sep 17 00:00:00 2001
From: "Tham, Mun Yew" <mun.yew.tham@intel.com>
Date: Tue, 4 Jun 2024 19:01:24 +0800
Subject: [PATCH 2/4] Revert "net: ethernet: altera-tse: Convert to mdio-regmap
 and use PCS Lynx"

This reverts commit db48abbaa18e571106711b42affe68ca6f36ca5a.
---
 drivers/net/ethernet/altera/Kconfig           | 2 --
 drivers/net/ethernet/altera/altera_tse_main.c | 3 ++-
 2 files changed, 2 insertions(+), 3 deletions(-)

diff --git a/drivers/net/ethernet/altera/Kconfig b/drivers/net/ethernet/altera/Kconfig
index 13829d3900a4..0995009d155f 100644
--- a/drivers/net/ethernet/altera/Kconfig
+++ b/drivers/net/ethernet/altera/Kconfig
@@ -16,8 +16,6 @@ config ALTERA_TSE
 	select PHYLIB
 	select PHYLINK
 	select PCS_ALTERA_TSE
-	select MDIO_REGMAP
-	select REGMAP_MMIO
 	help
 	imply PTP_1588_CLOCK
 	---help---
diff --git a/drivers/net/ethernet/altera/altera_tse_main.c b/drivers/net/ethernet/altera/altera_tse_main.c
index 57ac14799802..b54c13d8d29b 100644
--- a/drivers/net/ethernet/altera/altera_tse_main.c
+++ b/drivers/net/ethernet/altera/altera_tse_main.c
@@ -1471,6 +1471,7 @@ static int altera_tse_probe(struct platform_device *pdev)
 			 priv->tx_irq);
 
 	priv->pcs = alt_tse_pcs_create(ndev, priv->pcs_base, pcs_reg_width);
+
 	if (IS_ERR(priv->pcs)) {
 		ret = PTR_ERR(priv->pcs);
 		goto err_init_phy;
@@ -1521,6 +1522,7 @@ static int altera_tse_probe(struct platform_device *pdev)
 	}
 
 	return 0;
+
 err_init_phy:
 	unregister_netdev(ndev);
 err_register_netdev:
@@ -1544,7 +1546,6 @@ static int altera_tse_remove(struct platform_device *pdev)
 	altera_tse_mdio_destroy(ndev);
 	unregister_netdev(ndev);
 	phylink_destroy(priv->phylink);
-
 	free_netdev(ndev);
 
 	return 0;
-- 
2.25.1

